### PR TITLE
Adding selection field for keypair type

### PR DESCRIPTION
**Pull request by maayarosama**
### Description

Adding selection for key pair type in profile manager
Screenshot from 2023-12-18 16-12-50

### Changes

- Added selection for  key pair type
- Added a warning for the user
- Added key pair type in profile interface
- Updated get grid method 
### Related Issues

#1113
### Checklist

- [ ] Tests included
- [x] Build pass
- [ ] Documentation
- [x] Code format and docstrings
- [x] Screenshots/Video attached (needed for UI changes)

### CODE REVIEW

**Review by AhmedHanafy725 — COMMENTED** [review]
we need to re-validate the mnemonic if the keypair is changed as most probably the account will be created with a certain keypair type

**Comment by AlaaElattar** [comment]
- Validation should be reloaded.
- In that case, I've added mnemonic of type ed25519 and then chose it.

[Screencast from 12-19-2023 03:00:31 PM.webm](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/54944491/898bdf59-d761-4992-9585-2529f0e9c9f3)

**Comment by AhmedHanafy725** [comment]
not sure if we should move the new warning message to the top or keep it. But if we keep it, I think it should have some spacing after @ehab-hassan 

image

**Comment by AlaaElattar** [comment]
> * Validation should be reloaded.
> * In that case, I've added mnemonic of type ed25519 and then chose it.
> 
>  Screencast.from.12-19-2023.03.00.31.PM.webm

Please consider the same for `Activate Account` and `Create Account` buttons.

File: packages/playground/src/weblets/profile_manager.vue [hunk 1]
```diff
@@ -252,6 +267,9 @@
               <v-alert type="error" variant="tonal" class="mt-2 mb-4" v-if="loginError">
                 {{ loginError }}
               </v-alert>
+              <v-alert variant="tonal" type="warning" v-if="activeTab === 1">
```
Comment (AlaaElattar) [inline packages/playground/src/weblets/profile_manager.vue:270]: It feels more like an info not a warning , what do u think ?

**Comment by AlaaElattar** [comment]
Also as long as the user can't create account with ed25519 (just import pre existing account), I think w should say that somewhere.

**Comment by ehab-hassan** [comment]
> not sure if we should move the new warning message to the top or keep it. But if we keep it, I think it should have some spacing after @ehab-hassan
> 
> image

yes please let's keep it down and make sure to have the same margin's first warning message had.

**Review by AlaaElattar — APPROVED** [review]

File: packages/playground/src/weblets/profile_manager.vue [hunk 2]
```diff
@@ -462,6 +490,15 @@ watch(
...
 );
+watch(
+  () => keypairType.value,
+  async (value, oldValue) => {
+    if (value !== oldValue) {
+      mnemonicInput.value?.validate();
+    }
+  },
+  { deep: false },
+);
```
Comment (amiraabouhadid) [inline packages/playground/src/weblets/profile_manager.vue:501]: can we please update the message on line 567 to "Couldn't get the user twin for the provided mnemonic **and keytype**".

**Review by amiraabouhadid — APPROVED** [review]

**Review by AhmedHanafy725 — APPROVED** [review]

**Pull request merged**